docs/registerSwitch_addPorts.commands

```diff
@@ -0,0 +1,30 @@
+Commands to register a switch with HaaS
```

Comment:
Any reason not to move the rest of the information in this file to USING, and just delete it?